src/sections/Pricing/data.js

```diff
@@ -25,7 +25,7 @@ export const details = [
         enterprise: <GiCheckMark className="yes-icon" />,
       },
       {
-        feature: "Configurable OAuth callbacks",
+        feature: "Configurable OAuth callbacks ",
```

Comment:
Please do not change anything else in the code. Specifically, avoid making whitespace changes or other lint-related modifications.